generate-release-notes.py

```diff
@@ -28,14 +47,22 @@ def issues_in_milestone():
         if m.title == MILESTONE:
             # Found the milestone in this repo - look for issues (but only
             # ones that have been closed!)
+            print(f"  found milestone {m.title}")
             issues = []
-            for i in repo.get_issues(milestone=m, state="closed", labels=['release-note-required']):
-                pr = i.as_pull_request()
+            milestone_issues = repo.get_issues(
+                milestone=m, state="closed", labels=["release-note-required"]
+            )
+            for issue in milestone_issues:
+                pr = issue.as_pull_request()
                 if pr.merged:
                     # Filter out PRs which are closed but not merged.
-                    issues.append(i)
+                    issues.append(issue)
+                else:
+                    print(f"WARNING: {pr.number} is not merged, skipping")
```

Comment:
Does this need to be a warning? If the PR is closed but not merged then we've thrown it away, we probably don't need to be reminded of that at release time.

Reply:
this is to be able to warn about open issues attached to the milestone. I will update to only warn for those